cmd/mattermost/commands/config.go

```diff
@@ -121,10 +122,7 @@ func getConfigStore(command *cobra.Command) (config.Store, error) {
 		return nil, errors.Wrap(err, "failed to initialize i18n")
 	}
 
-	configDSN, err := command.Flags().GetString("config")
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to parse --config flag")
-	}
+	configDSN := viper.GetString("config")
```

Comment:
Awkwardly, `viper` doesn't /set/ the flag in `cobra`, but receives any values set there via the command line.

Reply:
Well that's annoying :P